website/docs/dev_api_cpp.md

```diff
@@ -0,0 +1,124 @@
+---
+id: dev_api_cpp
+title: AYON C++ API
+sidebar_label: C++ API
+description: Information about AYON C++ API
+toc_max_heading_level: 5
+---
+
+What is the C++ Language standard? it's ISO C++17 Standard
```

Comment:
Having this sentence on first line seems weird (out of context).

Reply:
https://github.com/ynput/ayon-documentation/pull/239/commits/b534281136c835274d3343536b40171be9f9bca0